src/table/io/FitsSchemaInputMapper.cc

```diff
@@ -708,9 +712,21 @@ bool endswith(std::string const &s, std::string const &suffix) {
     return s.size() >= suffix.size() && s.compare(s.size() - suffix.size(), suffix.size(), suffix) == 0;
 }
 
-// Replace the last n characters of a string with a new suffix string, returning the result as a new string
-std::string replaceSuffix(std::string const &s, std::size_t n, std::string const &suffix) {
-    return s.substr(0, s.size() - n) + suffix;
+bool hasInstFluxUnits(FitsSchemaItem const & item) {
```

Comment:
Maybe name this `isFluxField`, since it's doing more than just checking the units?

Reply:
How about `isInstFluxField`?

Reply:
That works.